index.html

```diff
@@ -9,27 +9,32 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 
     <link rel="stylesheet" href="./index.css" />
+    <script defer src="./index.js"></script>
     <title>Toyyib</title>
   </head>
   <body>
 
     <div class="wrapper">
+
         <div class="top-container">
     
             <div class="header">
     
                 <a href="#" class="left-text">My Logo</a>
     
-                <nav class="right-hamburger">
+                <nav class="right-hamburger" id="hamburger">
     
-                    <img src="./assets/Union.png" alt="">
+                    <img src="./assets/Union.png" alt="" >
     
                 </nav>
 
                 <nav class="nav-li">
-                    <a href="#">Portfolio</a>
-                    <a href="#">About</a>
-                    <a href="#">Contact</a>
+                    <div class="x-men"><span>X</span></div>
```

Comment:
- [ ] when clicking on the "X" the menu pop-up doesn't close, it only closes when clicking on one of the links or just under the "X" please fix this.